rules/Php81/NodeFactory/EnumFactory.php

```diff
@@ -91,14 +98,70 @@ private function createEnumCaseFromConst(ClassConst $classConst): EnumCase
         return $enumCase;
     }
 
-    private function createEnumCaseFromDocComment(PhpDocTagNode $phpDocTagNode): EnumCase
+    /**
+     * @param array<int|string, mixed> $mapping
+     */
+    private function createEnumCaseFromDocComment(PhpDocTagNode $phpDocTagNode, array $mapping = []): EnumCase
     {
         /** @var MethodTagValueNode $nodeValue */
         $nodeValue = $phpDocTagNode->value;
-
+        $enumValue = $mapping[$nodeValue->methodName] ?? $nodeValue->methodName;
         $enumName = strtoupper($nodeValue->methodName);
-        $enumExpr = $this->builderFactory->val($nodeValue->methodName);
+        $enumExpr = $this->builderFactory->val($enumValue);
 
         return new EnumCase($enumName, $enumExpr);
     }
+
+    /**
+     * @return array<int|string, mixed>
+     */
+    private function generateMappingFromClass(Class_ $class): array
+    {
+        $mapping = [];
```

Comment:
this can be moved after next if.

Reply:
Ive just committed some changes to address your comments. Thanks.